cocos/2d/CCSpriteFrameCache.cpp

```diff
@@ -389,6 +389,10 @@ void SpriteFrameCache::addSpriteFramesWithFile(const std::string& plist)
 
         // remove .xxx
         size_t startPos = texturePath.find_last_of('.'); 
+        if(startPos != string::npos)
+        {
+            texturePath = texturePath.erase(startPos);
+        }
```

Comment:
It will erase twice?

Reply:
sorry i forgot it, it was fixed in the last commit

Reply:
> It will erase twice?

sorry i forgot it, it was fixed in the last commit